src/NServiceBus.AcceptanceTests/Satellites/When_a_message_is_available.cs

```diff
@@ -47,8 +47,8 @@ public MySatelliteFeature()
 
                 protected override void Setup(FeatureConfigurationContext context)
                 {
-                    var satelliteLogicalAddress = context.Settings.LogicalAddress().CreateQualifiedAddress("MySatellite");
-                    var satelliteAddress = context.Settings.GetTransportAddress(satelliteLogicalAddress);
+                    var satelliteLogicalAddress = context.Transport.LogicalAddress.CreateQualifiedAddress("MySatellite");
+                    var satelliteAddress = context.Transport.TransportInfrastructure.ToTransportAddress(satelliteLogicalAddress);
```

Comment:
I thin the `TransportInfrastructure `is enough since it is a well defined component